### PR TITLE
Adds documentation regarding the default branch.

### LINKED ISSUE #512: Document default branch is canon
This will probably be best in the Contributing.md in the org level files, but we could include a note somewhere in the docs as well.

### PR DESCRIPTION
**Pull request by pathunstrom**
Closes #512

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -71,8 +71,9 @@ You are welcome to work on any issue you would like, we just think those
 are a good starting place for those that are unfamiliar with the code base.
 
 Now you need to [fork ppb][fork]. Once you've done that, go through
-either column and find a ticket you like. Start a branch, add your name
-to the [CONTRIBUTORS file][contributors], commit that change and open a PR:
+either column and find a ticket you like. Create a branch from the HEAD of
+the canon branch, add your name to the [CONTRIBUTORS file][contributors],
```
Comment (AstraLuma) [inline CONTRIBUTING.md]:
```suggestion
the ``canon`` branch, add your name to the [CONTRIBUTORS file][contributors],
```

**Review by AstraLuma — APPROVED** [review]
Looks good, one formatting suggestion.

**Comment by pathunstrom** [comment]
There, accepted (I should not do markup past midnight.)

**Comment by AstraLuma** [comment]
same tho

bors r+

**Pull request merged**